docs/integrations/test-explorer.md

```diff
@@ -0,0 +1,71 @@
+---
+id: test-explorer
+title: Test Explorer
+---
+
+Test Explorer is under active development, this page explain how it works under the hood.
+
+## Test Discovery
```

Comment:
One other thing that might be helpful to specify in here. For a client to use this, do they just set `testExplorerProvider` to true and then call it a day? Which approach that you outline is the default? Or will they both work by default?

Reply:
I've added a paragraph that should address these concerns, will it be good enough?

Reply:
Hey! Yup, that's helpful, thanks!